components/i2c/sgp30/definition.json

```diff
@@ -0,0 +1,16 @@
+{
+  "displayName": "SGP30",
+  "i2cAddresses": [ "0x58" ],
+  "subcomponents": [
+    {
+      "displayName": "eCO2",
+      "sensorType": "co2",
+      "defaultPeriod": 900
+    },
+    {
+      "displayName": "TVOC",
+      "sensorType": "unspecified",
```

Comment:
@lorennorman  please see - we need to add "tvoc" (lcase) as a `sensorType`

Borrowing from the [CircuitPython Sensor Units](https://docs.circuitpython.org/en/latest/docs/design_guide.html#sensor-properties-and-units), let's call it "tvoc", it's a float and writes the "Total Volatile Organic Compounds in ppb".